recipes/torchsparse/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "torchsparse" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mit-han-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6fa80a875eb2bfa74a69dd1b8e64df1b699710d07c072ca504d4f25fbbcb8730
+
+build:
+  number: 0
+  skip: true                                 # [osx or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+  missing_dso_whitelist:
+    - "$RPATH/libc10.so"
+    - "$RPATH/libtorch_cpu.so"
+    - "$RPATH/libtorch_python.so"
+    - /lib/libstdc++.so.6
+    - /lib*/libgcc_s.so.1
+    - /lib*/libc.so.6
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
+
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}                 # [linux64 and cuda_compiler_version not in (undefined, 'None')]
+  host:
+    - python
+    - pip
+    - sparsehash
+    - pytorch-cpu                            # [osx or cuda_compiler_version in (undefined, 'None')]
+    - pytorch-gpu                            # [cuda_compiler_version not in (undefined, 'None')]
+  run:
+    - python
+    - {{ pin_compatible('pytorch-cpu') }}    # [osx or cuda_compiler_version in (undefined, 'None')]
```

Comment:
Does pytorch have a stable ABI between minor versions? I suspect you need `max_pin="x.x"` or `max_pin="x.x.x"`.

Reply:
Unfortunately, it sounds like the ABI is generally fairly unstable. Here is a brief thread discussing this: https://github.com/conda-forge/pytorch-cpu-feedstock/issues/40#issuecomment-803133327 -- what are your thoughts?

Reply:
I would be tempted to go with `max_pin="x.x.x"` as building pytorch is enough of a pain that builds aren’t that frequent.